drivers/serial/Kconfig.nrfx

```diff
@@ -18,18 +18,35 @@ menuconfig UART_NRFX
 
 if UART_NRFX
 
+
+# ----------------- port 0 -----------------
 choice
 	prompt "UART Port 0 Driver type"
+	optional
 
 config UART_0_NRF_UART
 	bool "nRF UART 0"
 	depends on HAS_HW_NRF_UART0
 	select NRF_UART_PERIPHERAL
 	help
 	  Enable nRF UART without EasyDMA on port 0.
+
```

Comment:
Would it make sense to make comments regarding the flags?
HAS_HW_NRF_UART0 --- the hardware supports UART0
UART_0_NRF_UART  --- the user selected to use that UART0
NRF_UARTE_PERIPHERAL --- the actual selection of the driver and options for the driver
-- Just to make reading of this file easier.

Reply:
I do not want to introduce this change. I would like to be consistent with other drivers like SPI, SPIM, TWI or TWIM.

Reply:
I think this is pretty straight-forward; the naming conventions are generic enough to be easily understood.